config/vendor-api-v1.yml

```diff
@@ -764,6 +765,12 @@ components:
           description: The candidate’s experience as a volunteer
           items:
             $ref: "#/components/schemas/WorkExperience"
+        breaks:
```

Comment:
Because the other keys here are arrays and "breaks" is plural, I would've expected this to be an array too. Maybe `work_history_break_explanation` or something?

Reply:
Yes